forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/repository/migration/v3_1_6/MongoDbPaymentsChangeLog.java

```diff
@@ -62,7 +62,8 @@ public void migratePaymentDocuments(MongoTemplate mongoTemplate) {
         log.info("-----------------------------------------------------------------------");
         log.info("Migrating Payments API data from v3.1.6 to v3.1.6 patch...");
 
-        int docsUpdated = getObjectsToUpdate().stream().mapToInt(objectToUpdate -> upgrade(mongoTemplate, objectToUpdate.legacyClass, objectToUpdate.writeRiskParentField)).sum();
+        int docsUpdated = getObjectsToUpdate().stream().mapToInt(
```

Comment:
Probably will be better a rollback of this file.

Reply:
Just formatting.